connectors/es/client.py

```diff
@@ -41,6 +46,10 @@ def __init__(self, config):
             use_default_ports_for_scheme=True,
         )
         self._sleeps = CancellableSleeps()
+        self._retrier = TransientElasticsearchRetrier(
+            logger, config.get("max_retries", 5), config.get("retry_timeout", 10)
```

Comment:
I prefer to use constants like `DEFAULT_MAX_RETRIES` instead of the raw number. It's just easier to find and change them.

Reply:
Added constants in https://github.com/elastic/connectors/pull/2063/commits/a4d4f8bd1b15113d12f119920ea5ecb0753b48a7